bin/webpack/createConfig.js

```diff
@@ -2,7 +2,9 @@ function createConfig(options) {
   const path = require("path");
   const webpack = require("webpack");
   const WebpackNotifierPlugin = require("webpack-notifier");
+  var HappyPack = require('happypack');
```

Comment:
yea i was skeptical as well but it was super easy to set up and cut the build time in half for me. seems good so far